ui/00_message/common/item_name.xml

```diff
@@ -19659,7 +19659,7 @@
   </item>
   <item id="7839">
     <original>鬼どぶろく</original>
-    <translation>Demon Unrefined Wine</translation>
+    <translation>Unrefined Demon Wine</translation>
```

Comment:
You're certain this is an error? I try to match the DDON translation wikidot for easier end-user use.